### PR TITLE
Embed narrative into app

### PR DESCRIPTION
**Pull request by lee-vivian**
- Embedded the narrative into the app in the form of 2 expanders (one for socioeconomic analysis, another for tweets analysis)
- Added interactivity so that the relevant features are auto-loaded and expanders opened when the user clicks on the specified sidebar radio button as they follow along with the narrative
- Made some minor wording changes to the narrative

Maybe pull the branch locally and do a run-through of the narrative as if you're the end user. Let me know if anything should be added, removed or changed. Thanks!

### CODE REVIEW

File: streamlit_app.py [hunk 1]
```diff
@@ -460,6 +475,72 @@ def draw_embedded_tweets(state, container):
...
+    container.write("We hope you found this narrative compelling, or at the very least interesting. We have only scratched the surface of some of the popular opinions that can be explored and insights that can be drawn from our app. ")
+    container.write("We encourage you to ask your own questions and interact with our app further to see what new conclusions can be drawn from exploring the data. Some ideas include: What impact did the heat and summer months have on the number of cases? And is there a correlation between education level and the decision to wear a mask while out in public? See what myths and common beliefs you can help support or debunk.")
+    container.write("Happy Busting!  \n - Mythbusters")
```
Comment (jkosh44) [inline streamlit_app.py]: I think this get's rendered as markdown which means that "- Mythbusters" will look like
- Mythbusters

**Comment by jkosh44** [comment]
I'm not sure if this is related to the PR but I managed to break it
Screenshot from 2020-12-08 11-47-43

File: streamlit_app.py (hunk 1 above)
Comment (lee-vivian) [reply streamlit_app.py]: Thanks! I changed it to italics now so it'll look like:

Happy Busting!
*Mythbusters*

**Comment by jkosh44** [comment]
> I'm not sure if this is related to the PR but I managed to break it
> Screenshot from 2020-12-08 11-47-43

Seems to be that if anything but "Manual" or "Narrative Population" is selected and you select "Population" as the indicator, then it will error out.

**Comment by lee-vivian** [comment]
Oh my bad. I think I know what's causing that. Good catch!

**Review by jkosh44 — APPROVED** [review]

**Pull request merged**